### ISSUE TITLE
Does this fork support @prisma/client@3.0.2?

### ISSUE THREAD

**Issue by nrgapple** [kenchi/nexus-plugin-prisma#2]

**Comment by deadcoder0904** [comment]
Nope. I am trying to do the same thing & I get this error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: trendx@1.0.0
npm ERR! Found: @prisma/client@3.0.2
npm ERR! node_modules/@prisma/client
npm ERR!   @prisma/client@"3.0.x" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @prisma/client@"2.30.x" from @kenchi/nexus-plugin-prisma@0.37.1
npm ERR! node_modules/@kenchi/nexus-plugin-prisma
npm ERR!   @kenchi/nexus-plugin-prisma@"*" from the root project
npm ERR!
```

@kenchi any reason it doesn't support the latest version?

**Comment by oceandrama** [comment]
@bkrausz Hi! Thanks for your fork! Do you have plans to support Prisma v3?

**Comment by bkrausz** [comment]
I haven't had a chance to update it, but will try to do so this week.

**Comment by bkrausz** [comment]
I just published 3.1.x support. The only tweak necessary was to support the [new Json null handling](https://www.prisma.io/docs/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3#json-null-equality). It's possible (likely) that null filtering on Json fields does not work, but everything else should.
